promgen/templates/promgen/permission_row.html

```diff
@@ -20,6 +20,21 @@
       {{ perm|upper }}
     </span>
     {% endfor %}
+
+    <button class="btn btn-xs" data-toggle="collapse" data-target="#user{{user.id}}">
+      <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span>
```

Comment:
I think the refresh icon is not very appropriate. Since we are going to change a setting (in this case, the role of a member), I think it would be better if we use the cog icon. Please try it:

```suggestion
      <span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
```